client/js/components/header/burger.js

```diff
@@ -0,0 +1,61 @@
+import showHide from './../show-hide';
+import { KEYS } from './../../util';
+
+const headerBurger = (el) => {
+  const headerLower = el.querySelector('.js-header-lower');
+  const burger = showHide({el: headerLower});
+
+  const init = () => {
+    setBurgerAria(isBurgerVisible());
+    handleEvents();
+  };
+
+  const setBurgerAria = (value) => {
+    if (value) {
+      burger.el.setAttribute('aria-expanded', burger.getActive());
+    } else {
+      burger.el.removeAttribute('aria-expanded');
+    }
+  };
+
+  const isBurgerVisible = () => {
+    const isDisplayed = window.getComputedStyle(burger.trigger)
```

Comment:
This is as separate from the `aria-expanded` attribute – it instead determines whether the burger menu is currently displayed (i.e. is the viewport 'mobile').

On 'desktop', the main navigation links are always visible, so they don't have a require a concept of `aria-expanded` (it wouldn't make sense for a screenreader to read 'expanded' or 'collapsed' for UI that isn't expand-/collapsible.

This isn't the case on 'mobile' – so we need to determine whether or not the component should have an `aria-expanded` attribute (given the mobile and desktop versions share the same markup).